.gitignore

```diff
@@ -114,7 +114,7 @@ tests/fsharp/typecheck/sigs/neg94-pre.dll
 times
 /tests/fsharpqa/testenv/bin/System.ValueTuple.dll
 source_link.json
-.vs/
+.service/
```

Comment:
This is an accident; `.vs/` needs to be added back here

Reply:
Good catch! :)